contracts/peaks/BadgerSettPeak.sol

```diff
@@ -41,6 +44,21 @@ contract BadgerSettPeak is AccessControlDefended, IBadgerSettPeak {
         core = ICore(_core);
     }
 
+    modifier onlyGuardianOrGovernance() {
+        require(msg.sender == guardian || msg.sender == owner(), "onlyGuardianOrGovernance");
+        _;
+    }
+
+    // ===== Pausing Functionality =====
+    
+    function pause() external onlyGuardianOrGovernance {
+        _pause();
+    }
+
+    function unpause() external onlyGovernance {
+        _unpause();
+    }
+
```

Comment:
Would prefer these be moved to `AccessControlDefended` to avoid code duplication across peaks.